todo_app/test_view_model.py

```diff
@@ -0,0 +1,58 @@
+from todo_app.data.view_model import ViewModel
+from todo_app.item import Item
+
+
+def test_todo_items_property_only_shows_todo_items_and_nothing_else() :
+   # ARRANGE
+    items = [
+        Item(1, "started Todo", "To Do"),
+        Item(2, "In Progress Todo", "Doing"),
+        Item(3, "Finished Todo", "Done")    
+    ]
```

Comment:
Really minor, but notice that the start of your tests is very similar in all three tests. We could look to de-duplicate that with a pytest fixture, which might look like:
```python
import pytest

@pytest.fixture
def items():
    return [
        Item(1, "started Todo", "To Do"),
        Item(2, "In Progress Todo", "Doing"),
        Item(3, "Finished Todo", "Done")    
    ]

def test_todo_items_property_only_shows_todo_items_and_nothing_else(items):
   # ARRANGE
    view_model = ViewModel(items)
    ....
```